packages/dd-trace/src/appsec/index.js

```diff
@@ -1,8 +1,51 @@
 'use strict'
 
-function enable () {
+const fs = require('fs')
+const path = require('path')
+const log = require('../log')
+const RuleManager = require('./rule_manager')
+const { INCOMING_HTTP_REQUEST_START } = require('../gateway/channels')
+const { startContext, propagate } = require('../gateway/engine/index')
+const Addresses = require('./addresses')
+
+function enable (config) {
+  try {
+    // TODO: enable dc_blocking: config.blocking === true
+
+    let rules = fs.readFileSync(path.join(__dirname, 'recommended.json'))
```

Comment:
a `require` would work here I think

Reply:
Yes I used this before, but according to bengl, require and json is a bad idea starting from node 12. It doesn't break right away but we should move away from it for good measure.

Reply:
lgtm :)